### ISSUE TITLE
[CI] Prevent suffixed crate versions

### ISSUE THREAD

**Issue by ggwpez** [paritytech/polkadot-sdk#3338]
In a similar spirit as https://github.com/paritytech/polkadot-sdk/issues/3293 i think we should prevent versions having a suffix.  
So that all versions need to be of the form `x.y.z` so that our semver tooling can operate on it.

**Comment by liamaharon** [comment]
Correct, for packages in `stable`. In `master` they should have suffixes if they have changes relative to stable.

**Comment by ggwpez** [comment]
Okay I see. That will be a bit difficult to spot and for now i would add the check it its simplest form.
